cabby/rvs/speak.py

```diff
@@ -16,14 +16,64 @@
 
 from typing import Text
 
-def describe_route(pivot_poi: Text, goal_poi: Text) -> Text:
+from cabby.geo import directions
+
+# Use alias for Direction to keep code cleaner.
+Direction = directions.Direction
+
+def describe_meeting_point(
+  pivot: Text, goal: Text,
+  bearing_pivot_goal: float, distance_pivot_goal: float) -> Text:
   '''Preliminary example template for generating an RVS instruction.
   
   Arguments:
-    pivot_poi: The POI used to orient with respect to the goal.
-    goal_poi: The POI that is the intended meeting location.
+    pivot: The POI used to orient with respect to the goal.
+    goal: The POI that is the intended meeting location.
+    bearing_pivot_goal: The relative bearing of the goal when heading to the
+      pivot from the start point.
+    distance_pivot_goal: The distance from the pivot to the goal.
   Returns:
     A string describing the goal location with respect to the reference.
   
   '''
-  return f'go to the {goal_poi} near the {pivot_poi}.'
```

Comment:
missing newline

Reply:
I don't want to return newline here -- the caller may be piecing together multiple such instructions without newlines.